vault/identity_store_oidc_test.go

```diff
@@ -937,12 +937,32 @@ func TestOIDC_PeriodicFunc(t *testing.T) {
 				numPublicKeys int
 			}{
 				{1, 2, 2},
-				{2, 3, 3},
-				{3, 3, 3},
-				{4, 3, 3},
-				{5, 3, 3},
-				{6, 3, 3},
-				{7, 3, 3},
+				{2, 2, 2},
```

Comment:
How come the values got bumped down by 1 after this change?

Reply:
This test is flaky on test-go-nightly-32bit. I took the opportunity to address the issue since I am touching these tests again.

I think this is due to the short 2 second cycle period in which we call the test. Key deletions sometimes happen before or after a given cycle. So instead of expecting exactly how many keys we have at a given cycle, I am testing for a threshold of keys.

There may be a better way to go about this. But I didn't want to increase the test time too much. Now that I am thinking about it here maybe I should not have included this in this PR. :/